### PR TITLE
[terra-avatar] Migrate terra-avatar docs

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary

This pr migrates terra avatar docs to terra-core-docs.

Closes #

### Deployment Link

https://terra-core-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "clean:obsolete-snapshots": "npm test -- -u",
     "create-props-markdown": "node scripts/props-table-gen/generatePropsTables.js",
     "compile": "npm run compile:clean && npm run compile:build",
-    "compile:build": "lerna ls -a | sed 's/@cerner\\///g' | cut -d ' ' -f1 | xargs -I {} babel packages/{}/src --out-dir packages/{}/lib --copy-files",
+    "compile:build": "lerna run compile --parallel",
```
Comment (mjhenkes) [inline package.json:43]: I changed this to the style that we use in terra-toolkit, it saves about a minute of time locally, though uses more cpu. (A fine trade off imo)

Before:
npm run compile:build  93.07s user 12.92s system 103% cpu 1:42.69 total

After:
npm run compile:build  133.44s user 23.11s system 371% cpu 42.142 total

Comment (mjhenkes) [reply package.json:43]: It's also more colorful, which everyone can appreciate

**Review by tbiethman — APPROVED** [review]

**Review by pranav300 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (benbcai) [reply package.json:43]: We should also do the same for framework and clinical next time.

File: packages/terra-core-docs/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Migrate doc to terra-avatar
```
Comment (benbcai) [inline packages/terra-core-docs/CHANGELOG.md:6]:
```suggestion
  * Migrate doc from terra-avatar
```

**Review by benbcai — APPROVED** [review]

File: packages/terra-core-docs/CHANGELOG.md (hunk 2 above)
Comment (sdadn) [reply packages/terra-core-docs/CHANGELOG.md]: Shouldn't this also be in past tense? `Migrated`?

**Pull request merged**